anycloud/cli/src/main.rs

```diff
@@ -61,40 +67,50 @@ pub async fn main() {
       )
     );
 
-  authenticate().await;
   let matches = app.get_matches();
   match matches.subcommand() {
     ("new", Some(matches)) => {
-      let dockerfile_b64 = get_dockerfile_b64().await;
-      let app_tar_gz_b64 = get_app_tar_gz_b64(true).await;
-      let env_b64 = match matches.value_of("env-file") {
-        Some(env_file) => Some(get_env_file_b64(env_file.to_string()).await),
-        None => None,
+      let non_interactive: bool = match matches.values_of("NON_INTERACTIVE") {
+        Some(_) => true,
+        None => false,
       };
-      deploy::new(
+      authenticate(non_interactive).await;
+      new_or_upgrade(
+        "new",
         anycloud_agz,
-        Some((dockerfile_b64, app_tar_gz_b64)),
-        env_b64,
+        matches.value_of("env-file"),
+        matches.value_of("app-name"),
+        matches.value_of("config-name"),
+        non_interactive,
       )
       .await;
     }
-    ("terminate", _) => deploy::terminate().await,
+    ("terminate", _) => {
+      authenticate(false).await;
```

Comment:
Should we have a `TODO` comment here to allow non-interactive termination in the future?

A normal CI/CD usage wouldn't need it, but if you had a test that fully spun up and shut down a test service, it would be necessary.

Reply:
Oh yes, forgot to implement it actually, will add this implementation in this PR